codelists/models.py

```diff
@@ -372,7 +375,9 @@ def coding_system_id(self):
 
     @cached_property
     def coding_system(self):
-        return CODING_SYSTEMS[self.coding_system_id]
+        return self.codelist.coding_system_cls(
+            database_alias=self.coding_system_version_slug
```

Comment:
Agreed; I hadn't removed the `db_name` property because it would have meant changing code that wasn't related to this PR, but it isn't used anywhere else now.  I'd lean towards `database_alias` because it's more explicit.